Example/Podfile

```diff
@@ -1,9 +1,12 @@
+platform :ios, '9.0'
 use_frameworks!
 
 target 'GASocialLogin_Example' do
   
   # pod 'GoogleSignIn' , '~> 5.0.2'
   # pod 'FBSDKLoginKit', '~> 7.1.1'
+  
+  pod 'GASocialLogin', :path => '../'
```

Comment:
please explain this line

Reply:
The :path option in a CocoaPods podspec is used to specify the location of the source code for a dependency. 
In this case, :path => '../' means that the source code for the GASocialLogin dependency is located one directory up from the directory where the Podfile is located. 
The '..' notation is a relative file path that refers to the parent directory of the current directory. 
So, when CocoaPods installs the GASocialLogin dependency, it will look for the source code in the directory located one level above the directory containing the Podfile.

Anyway this line should be ignored (discarded).